crates/shielder_bindings/.cargo/config.toml

```diff
@@ -0,0 +1,47 @@
+[target.wasm32-unknown-unknown]
```

Comment:
since this is a copy of `crates/shielder-wasm/.cargo/config.toml`, we might use a symlink to maintain a single cargo configuration for wasm-related crates; optionally, we could just move this file somewhere and link to it from any crate that requires these compilation conf